packages/shared/auth0-client/src/index.ts

```diff
@@ -103,6 +126,24 @@ export default class Auth0Client {
     });
   }
 
+  async validateUserCredentials(username: string, password: string) : Promise<APIResponse<boolean>> {
+    const userFetchResult = await this.getUserByEmail(username);
+
+    if (userFetchResult.status != ResponseStatus.Success) {
```

Comment:
How do you feel about doing this check as part of `packages/apps/api/src/handlers/auth.ts#validateCredentials`? That way we keep this particular call specific to its purpose, and the Lambda Function can concern itself with checking if the user exists?

Reply:
yeah, I wasn't really sure about this initially, but keeping /auth strictly to auth seems nicer. It makes me less nervous about user enumeration even though it's a private API endpoint.